### DISCUSSION TITLE
Cubic controller decoder issues with 85.x firmware

### DISCUSSION THREAD

**Discussion by Katie3765** [udotdevelopment/ATSPM#223] [Q&A]
Has anyone else been having issues with decoding controller logs for Cubic controllers in the newer 85.x firmware? We've recently identified severe data quality issues with it. We've been informed by a Cubic representative that external software decoding of controller logs is no longer a supported process and that no decoder will be provided. Wondering if anyone else has encountered this or found a solution to decoding the newer files.

**Comment by ian-cameron** [comment]
Yes, 2.2.0 decoder version didn't work with 85.1, it was supposed to be for 85.3 I heard from a rep.  We successfully used decoder 2.0.1 for a custom version 85.1 though.  I really hope Cubic still support open source ATSPM by providing a functioning high resolution data decoder!
